### DISCUSSION TITLE
Play sound when espanso is triggered

### DISCUSSION THREAD

**Discussion by killtux** [espanso/espanso#1170] [Feature requests and Ideas]
i would be nice to have an option for a sound notification the moment espanso is triggered. for example like "beeftext" for windows does. it should be simple and not too distracting.

**Comment by federico-terzi** [comment]
Hey @killtux,

This could be a nice feature to have, I'll add it to the project roadmap :)
Do you have any suggestion for the sound?
If you could link me a good royalty free sound effect file, that could be a great help :)

Cheers

**Comment by killtux** [comment]
I think that would be pretty simple and nice: https://notificationsounds.com/notification-sounds/clearly-602

**Comment by federico-terzi** [comment]
> I think that would be pretty simple and nice: https://notificationsounds.com/notification-sounds/clearly-602

That would be a great candidate indeed. In the same website I also liked this one: https://notificationsounds.com/sound-effects/pull-back-325

I will probably add both of them and let the user decide.

Thanks for your help :)

**Comment by killtux** [comment]
any news on this? :)

**Comment by federico-terzi** [comment]
Hey @killtux,
Not much at the moment :) I want to add this feature, but I still have to evaluate the effects of having that additional dependency. So it will take a while :) I'll keep you posted

**Comment by bartv42** [comment]
I'd like to give my +1 to this as well :) Coming from Typinator on Mac, I've grown used to having the audible confirmation that my expansion worked. Their sound is an even more minimal 'tick'.

**Comment by federico-terzi** [comment]
@bartv42 Thank you for the feedback! I can guarantee that this will get implemented eventually, though I don't have any ETA yet :)

**Comment by andreystepanov** [comment]
Would be great if user could also specify path to mp3/wav (depends on the implementation) file in configuration.

Something along these lines:

```yaml
notification_sound = true
notification_sound_path = "/path/to/sound.mp3"
```

Where `notification_sound_path` by default (if not specified) would point to sound of your choice (in the comments above).

An interesting one from the same website: https://notificationsounds.com/sound-effects/armed-256

As a suggestion, i also like the sound from TextExpander, kind of a "bubble pop" sound, but need some time to find copyright-free alternative :)

**Comment by andreystepanov** [comment]
Found one on YouTube, just as an example:

https://www.youtube.com/watch?v=MN5wEWxyOCI

**Comment by federico-terzi** [comment]
@andreystepanov Thank you for the help!

**Comment by jczanfona** [comment]
So ... is it possible to make a sound when copying? <kbd>Ctrl</kbd>+<kbd>C</kbd>

**Comment by kudumi** [comment]
is this possible now?

**Comment by federico-terzi** [comment]
@kudumi Not yet :) There are a few other things we'll need to implement first

**Comment by AucaCoyan** [comment]
Created https://github.com/espanso/espanso/issues/2135 to track it
